buildbot_nix/nix_status_generator.py

```diff
@@ -131,26 +190,37 @@ async def generate(
                 formatter, master, reporter, data
             )
 
-            if event in {"started-nix-eval", "finished-nix-eval"}:
+            if event in {
```

Comment:
Would it not make sense to convert the string to the our event enum and than return early if the event is not in our enum?

Reply:
I believe enums have a __members__ field or something that can be checked against.

Reply:
Oh yeah, that makes way more sense, good catch

Reply:
Done, but untested